### DISCUSSION TITLE
Citra MMJ tendra Vulkan??

### DISCUSSION THREAD

**Discussion by LuisPerss** [weihuoya/citra#902] [Q&A]
Una pregunta señores:citra mmj tendra vulkan durante este año por decir alguna un año o fecha? Ya que en dipositivos mali esto seria muy veneficioso ya que el opengl esta optimizado por parte de los qualcoms y no mucho por parte de los exynos,kirim,mediatek y unisocts.

**Comment by AndresNorm** [comment]
No, and will not be in the Future
